src/game.zig

```diff
@@ -442,6 +455,27 @@ pub const Player = struct { // MARK: Player
 		inventory.placeBlock(selectedSlot);
 	}
 
+	pub fn damage(dam: f32, cause: DamageType) void {
```

Comment:
Please remove this function, all of its functionality should be done in Inventory.addHealth.